vagrant/README.md

```diff
@@ -75,6 +75,19 @@ From here you can decide what type of setup to run.
 
   1. https://whamcloud.github.io/Online-Help/docs/Contributor_Docs/cd_Managed_ZFS.html
 
+- Monitored Ldiskfs with LVM Metadata:
+
+  ```sh
+	vagrant provision --provision-with=install-ldiskfs-no-iml,configure-lustre-network,create-ldiskfs-lvm-fs,mount-ldiskfs-lvm-fs
+	```
+
+- Monitored Ldiskfs with LVM Metadata and HA:
+
+  ```sh
+	vagrant provision --provision-with=install-ldiskfs-no-iml,configure-lustre-network,create-ldiskfs-lvm-fs,ha-ldiskfs-lvm-fs-step1
+	vagrant provision --provision-with=ha-ldiskfs-lvm-fs-step2
+	```
```

Comment:
These don't provision the oss nodes to have lustre targets or to mount them. What should probably happen is the provisioners for ldiskfs are reused under the ldiskfs-lvm-* name for the osses so a whole filesystem can be provisioned.